### ISSUE TITLE
Possibility to leave a field empty

### ISSUE THREAD

**Issue by sandersantema** [glfharris/ankidown#7]
Currently adding a note like the following wouldn't be possible:

*Template:* 
```
# {Front}
# {Back}
# {Hint}
```

*Note:*
```
# Front Example
# Back Example
```

This would be good to have since you might for instance not want to add a hint to every note but use the same template non the less. 

This would hopefully also allow for the following:

*Template:* 
```
# Front
{Front}
# Back
{Back}
# Hint
{Hint}
# Source
{Source}
```

*Note:*
```
# Front
Front Example
# Back
Back Example
# Hint

# Source
Example Source
```

**Comment by glfharris** [comment]
I agree there's definitely a need for something like this, although there's a potential headache with the templating system. I think the easy way would just be to leave a blank line, which is enough to match as a field. Then do a cull on whitespace only fields in the note processing.
